Singer.API/ClientApp/src/app/modules/core/models/registrant.model.ts

```diff
@@ -1,12 +1,74 @@
-import { RegistrationStatus, EventRelevantCareUserDTO } from 'src/app/modules/core/models/singerevent.model';
+import {
+   RegistrationStatus,
+   EventRelevantCareUserDTO,
+   EventCareUserRegistrationDTO,
+} from 'src/app/modules/core/models/singerevent.model';
 import { CareUserDTO } from 'src/app/modules/core/models/careuser.model';
 export class Registrant {
+   registrationId: string;
    careUserId: string;
    name: string;
    registrationStatus: RegistrationStatus;
-   constructor(careUser: EventRelevantCareUserDTO, registrationStatus: RegistrationStatus) {
-      this.careUserId = careUser.id;
-      this.name = `${careUser.firstName} ${careUser.lastName}`;
+
+   constructor(
+      registrationId: string,
+      careUserId: string,
+      firstName: string,
+      lastName: string,
+      registrationStatus: RegistrationStatus
+   ) {
+      this.careUserId = careUserId;
+      this.registrationId = registrationId;
+      this.name = `${firstName} ${lastName}`;
       this.registrationStatus = registrationStatus;
    }
 }
+
+export class EventSlot {
```

Comment:
We already have a `export interface EventSlot` in `event-slot.ts`, why create a class with the same name here?
In my opinion an `EventSlot` definition (regardless if class or interface) does not belong in this file.
This is very risky because based on the import you could be using one or the other

Reply:
Agreed, will check for duplicate naming issues